src/AsteroidMining/SunStormTest.java

```diff
@@ -0,0 +1,83 @@
+package src.AsteroidMining;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.Random;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class SunStormTest {
+
+    @Test
+    void SettlerCollisionWith() {
+        Random rand = new Random();
```

Comment:
You can take out the Random object as the class member field, it is called multiple times in every test method. Same for Handler and Asteroid maybe, since you reuse them.